upgrade/helpers/tasks.py

```diff
@@ -161,7 +161,7 @@ def sync_capsule_repos_to_upgrade(capsules):
         id=max([cv_ver.id for cv_ver in cv.read().version])).read()
     logger.info("Content view {} promotion has started successfully".
                 format(cv.name))
-    published_ver.promote(data={'environment_id': lenv.id, 'force': False})
+    published_ver.promote(data={'environment_ids': lenv.id, 'force': False})
```

Comment:
I saw your comment that it works ! But `environment_ids` should accept list here right ? We are providing an int.

Reply:
Fixed.